js/main.js

```diff
@@ -1,2 +1,30 @@
 /* global data */
 /* exported data */
+var $form = document.querySelector('#entry-form');
+var $title = $form.elements.title;
+var $notes = $form.elements.notes;
+var $photoUrl = $form.elements.photoUrl;
+var $photo = document.querySelector('#photo');
+var n = 1;
```

Comment:
There should be no need for this variable, you should be using the `data.nextEntryId` property for this purpose.